content/scripts/build.js

```diff
@@ -368,27 +451,27 @@ class Builder {
     flawsByUri.set(uri, flaws);
   }
 
-  reportFlaws(type) {
-    const flawsByUri = this.flawsByType.get(type);
-    if (flawsByUri) {
-      if (flawsByUri.size) {
-        const header = `\nFlaws found within ${flawsByUri.size} document(s) while rendering macros`;
-        if (this.options.flaws === FLAWS_LEVELS.IGNORE) {
-          this.logger.info(chalk.yellow.bold(header + "."));
-        } else {
-          this.logger.error(chalk.red.bold(header + ":"));
-          for (const [uri, flaws] of flawsByUri) {
-            this.logger.error(
-              chalk.red.bold(`*** flaw(s) while rendering ${uri}:`)
-            );
-            for (const flaw of flaws) {
-              this.logger.error(chalk.red(`${flaw}\n`));
-            }
-          }
-        }
-      }
-    }
-  }
+  // reportFlaws(type) {
+  //   const flawsByUri = this.flawsByType.get(type);
+  //   if (flawsByUri) {
+  //     if (flawsByUri.size) {
+  //       const header = `\nFlaws found within ${flawsByUri.size} document(s) while rendering macros`;
+  //       if (this.options.flaws === FLAW_LEVELS.IGNORE) {
+  //         this.logger.info(chalk.yellow.bold(header + "."));
+  //       } else {
+  //         this.logger.error(chalk.red.bold(header + ":"));
+  //         for (const [uri, flaws] of flawsByUri) {
+  //           this.logger.error(
+  //             chalk.red.bold(`*** flaw(s) while rendering ${uri}:`)
+  //           );
+  //           for (const flaw of flaws) {
+  //             this.logger.error(chalk.red(`${flaw}\n`));
+  //           }
+  //         }
+  //       }
+  //     }
+  //   }
+  // }
```

Comment:
It probably needs to be deleted. I kept it for my sanity.

Reply:
Go ahead and delete it!